docs/windows/rnode.toml

```diff
@@ -10,6 +10,7 @@ bootstrap = "rnode://c61769b39d368cbcbc9499634e030386c79d5b02@52.119.8.108:40400
 standalone = true
 data-dir = "C:/RChain/data"
 map-size = 200000000
+in-memory-store = false
```

Comment:
this configuration file is for windows, should it not be set to true?

Reply:
On Windows (not WSL) LMDB works fine. The only problem we have now - absent native DLLs in our port of kalium. In memory store should be default for WSL, but we don't have a sample config for WSL yet.